.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly
+matrix:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true
+before_install:
+  # minifb requires x11 header files
+  - sudo apt-get update
+  - sudo apt-get install libx11-dev
```

Comment:
For apt packages use the apt add-on: https://docs.travis-ci.com/user/installing-dependencies/#adding-apt-packages